src/commands/player/FriendlyFightCommand.js

```diff
@@ -0,0 +1,9 @@
+module.exports.help = {
+	"name" : "friendlyfight"
```

Comment:
Codacy has a fix for the issue: [Extra space after key 'name'.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7575360)
```suggestion
	"name": "friendlyfight"
```